### PR TITLE
Do not send member updates to all

### PR DESCRIPTION
**Pull request by pcapriotti**
This PR changes which users receive the following team events:

- `team.update`
- `team.member-join`
- `team.member-leave`
- `team.member-update`

Without this PR these events are sent to arbitrary subset of 2000 team members. This is due to the fanout limit. For teams smaller than 2000 members all team members receive the events.

With this PR these events are sent only to team admins, so that the `team-settings` app can receive them. These events are sent transient, i.e. they are only sent to open websockets (opened by `team-settings`) and not persisted in the admin's notification stream.
This PR also limits the number of team admins of a team to 2000.
This PR also introduced an index of all team admins (table `galley.team_admin`), which is backfilled in an automated data migration.

This PR does not remove the fanout limit in general.

Tracked by https://wearezeta.atlassian.net/browse/WPB-2565

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: services/galley/schema/src/V82_CreateTableTeamAdmin.hs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+migration :: Migration
+migration = Migration 82 "Create table `team_admin`" $ do
+  schema'
+    [r|
+        CREATE TABLE team_admin (
+            team   uuid,
+            user   uuid,
+            PRIMARY KEY (team, user)
+        );
```
Comment (jschaul) [inline services/galley/schema/src/V82_CreateTableTeamAdmin.hs:35]: Please use LeveledCompactionStrategy, it's not cassandra's default, but for almost all kind of data we have at Wire it's the better choice.

```suggestion
        ) with compaction = {'class': 'LeveledCompactionStrategy'};
```

**Review by jschaul — APPROVED** [review]
I just reviewed cassandra tables; data migration; cql statement use. Looks good! Minor nit-pick inline.

As for notifying users - behaviour was changed, is there a way to add a test to ensure the team admins who are online see the events? Or is there already a test covering that?

File: services/galley/schema/src/V82_CreateTableTeamAdmin.hs (hunk 1 above)
Comment (smatting) [reply services/galley/schema/src/V82_CreateTableTeamAdmin.hs]: Thanks! Updated schema https://github.com/wireapp/wire-server/pull/3431/commits/05ff0045c429763f797e46ebbe101dded772705f

**Comment by smatting** [comment]
Integration tests passed for https://github.com/wireapp/wire-server/pull/3431/commits/b8032600353ecf0bd6ad7daf7efdf9190f500594

**Pull request merged**